vercel.json

```diff
@@ -2,22 +2,22 @@
   "redirects": [
     {
       "source": "/releases/mainnet/active/contracts.json",
-      "destination": "/releases/mainnet/1.9.1/contracts.json",
+      "destination": "/releases/mainnet/1.9.2/contracts.json",
       "permanent": false
     },
     {
       "source": "/releases/mainnet/active/",
-      "destination": "/releases/mainnet/1.9.1/index.html",
+      "destination": "/releases/mainnet/1.9.2/index.html",
```

Comment:
Same comment as above

Reply:
yes, just waited for the PIP addresses from deploy this time.